### PR TITLE
if we get an out of space exception and there is nothing is storage t…

### PR DESCRIPTION
**Pull request by stevejhiggs**
…hen storage does not work

### CODE REVIEW

File: lscache.js [hunk 1]
```diff
@@ -64,12 +64,16 @@
...
 
+    if (!localStorage) {
+      return false;
+    }
+
     try {
       setItem(key, value);
       removeItem(key);
       cachedStorage = true;
     } catch (e) {
```
Comment (pamelafox) [inline lscache.js]: Can you move the comment so it's above line 76 so we can truncate it to 80 characters per line?

**Comment by pamelafox** [comment]
Thanks for the PR, left one comment.

**Comment by stevejhiggs** [comment]
done

**Comment by pamelafox** [comment]
Great, merged!

**Pull request merged**